### ISSUE TITLE
The token '&&' is not a valid statement separator in this version.

### ISSUE THREAD

**Issue by ividrine** [aslamanver/jar-builder#1]
Getting this error when running build. I do get a notification saying jar was built successfully, but it isn't in my workspace.

I have powershell version 5.1.18362.145 set as my default terminal running in vs code version 1.42

Heres the full message

```
The token '&&' is not a valid statement separator in this version.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine
```

**Comment by gasser707** [comment]
getting the same problem !

**Comment by goldfishrock** [comment]
Me too. 

VSCode 1.43.0, 
Powershell 5.1.18362.628

**Comment by RodoHS** [comment]
Me too.

**Comment by Noah670** [comment]
Dude fix it

**Comment by aslamanver** [comment]
It's fixed now in 1.1.0

**Comment by aslamanver** [comment]
> Getting this error when running build. I do get a notification saying jar was built successfully, but it isn't in my workspace.
> 
> I have powershell version 5.1.18362.145 set as my default terminal running in vs code version 1.42
> 
> Heres the full message
> 
> ```
> The token '&&' is not a valid statement separator in this version.
>     + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
>     + FullyQualifiedErrorId : InvalidEndOfLine
> ```

It's appreciated.
